lib/email/receiver.rb

```diff
@@ -552,8 +552,13 @@ def previous_replies_regex
       @previous_replies_regex ||= /^--[- ]\n\*#{I18n.t("user_notifications.previous_discussion")}\*\n/im
     end
 
+    def reply_above_line_regex
+      @reply_above_line_regex ||= /\n#{I18n.t("user_notifications.reply_above_line")}\n/im
```

Comment:
Can we somehow get all the translations for this string and generate a regexp from that? (same for "previous_replies")

Reply:
I followed up in:

https://github.com/discourse/discourse/pull/14081